src/components/HeartbeatBar.vue

```diff
@@ -91,14 +75,14 @@ export default {
 
             let width
             if (this.maxBeat > 0) {
-                width = (this.beatWidth + this.beatMargin * 2) * this.maxBeat + (leftRight * 2) + "px"
+                width = `${(this.beatWidth + this.beatMargin * 2) * this.maxBeat + (leftRight * 2)}px`
```

Comment:
I also prefer not to strict to a string concatenation style. It is an example of bad use of ${}.  It is harder to read and harder to write than the "Plus" sign.

Reply:
I commented the `'prefer-template': 'error',` rule